scripts/wheel/rafscript.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Create the build directory
+git clone https://github.com/awslabs/raf --recursive
+cd raf
+export RAF_HOME=$(pwd)
+mkdir $RAF_HOME/build
+
+# Run the codegen for auto-generated source code
+bash ./scripts/src_codegen/run_all.sh
+
+# Configuration file for CMake
+cd $RAF_HOME/build
+cp ../cmake/config.cmake .
+
+# Edit the configuration file
```

Comment:
Add a todo to make this part configurable. In the future we are going to take arguments from command line to determine the configuration.